cirq/google/gate_sets.py

```diff
@@ -111,7 +111,7 @@
 }
 
 document(NAMED_GATESETS,
-         """"A convenience mapping from gateset names to gatesets""")
```

Comment:
how did this ever work?

Reply:
It just adds the quote to the front, so the doc string was
```"A convenience mapping from gateset names to gatesets```

Reply:
ah. I was way overthinking it